CastleNathria/Locales/deDE.lua

```diff
@@ -50,12 +50,12 @@ if L then
 
 	L.custom_off_select_boss_order = "Tötungsreihenfolge markieren"
 	L.custom_off_select_boss_order_desc = "Markiert die Reihenfolge, in welcher der Schlachtzug die Bosse tötet mit Kreuz {rt7}. Benötigt Schlachtzugsleiter oder Assistent zum Markieren."
-	L.custom_off_select_boss_order_value1 = "Niklaus -> Frieda -> Stavros"
-	L.custom_off_select_boss_order_value2 = "Frieda -> Niklaus -> Stavros"
-	L.custom_off_select_boss_order_value3 = "Stavros -> Niklaus -> Frieda"
-	L.custom_off_select_boss_order_value4 = "Niklaus -> Stavros -> Frieda"
-	L.custom_off_select_boss_order_value5 = "Frieda -> Stavros -> Niklaus"
-	L.custom_off_select_boss_order_value6 = "Stavros -> Frieda -> Niklaus"
+	-- L.custom_off_select_boss_order_value1 = "Niklaus -> Frieda -> Stavros"
```

Comment:
Why did you comment these out?

Reply:
As they are exactly the same as the English strings, I thought it would be better to not load them again because they are already loaded in enUS and in my understanding overwritten by deDE if available. So I always comment out translations that are not needed.